libraries/AP_Scripting/drivers/LTE_modem.md

```diff
@@ -188,3 +194,11 @@ Common issues:
 
 If the connection fails or is lost, the driver will automatically attempt
 to reconnect by restarting the connection sequence.
+
+# Logging
+
+A LTE log message is saved in the onboard log. That has signal
+strength information and data transfer statistics.
+
+A NAMED_VALUE_FLOAT MAVLink message "LTE_RSSI" is sent with the RSSI
```

Comment:
Have you considered using this packet?
https://mavlink.io/en/messages/common.html#RADIO_STATUS

Reply:
I did consider it, but it isn't a good fit